en/contributing/README.md

```diff
@@ -4,3 +4,27 @@ MAVSDK welcomes contributions!
 If you want to help or have suggestions/bug reports [please get in touch with the development team](../README.md#getting-help).
 
 This section contains everything you need to contribute, including topics about building the SDK from source code, running our integration and unit tests, and all other aspects of core development.
+

```

Comment:
I appreciate that this is mostly about C++ but I THINK that it would be best to put this doc as a peer of C++, Python, etc in the hierarchy. You would note the types of contribution possible for each language somewhere here.

Reply:
But it is now a peer and not in C++?

Reply:
Right now it is in C++. I am suggesting it is better the same level as Python, C++ etc:

![image](https://user-images.githubusercontent.com/5368500/112268866-b8b43680-8ccb-11eb-9bcb-450bff80290c.png)

Reply:
That's where it is now.